icu4c/source/config/dist-data.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+# Copyright (C) 2020 and later: Unicode, Inc. and others.
+
+# set VERSION to the ICU version. set top_srcdir to the parent of icurc
+# Note: You need to set LD_LIBRARY_PATH/etc before calling this script.
+export LD_LIBRARY_PATH=./lib:${LD_LIBRARY_PATH-/lib:/usr/lib:/usr/local/lib}
+export DYLD_LIBRARY_PATH=./lib:${DYLD_LIBRARY_PATH-/lib:/usr/lib:/usr/local/lib}
+
+if [ ! -d "${top_srcdir}" ]
+then
+    echo >&2 "$0: please set “top_srcdir” to the icu/icu4c/source dir"
```

Comment:
nit: are smart quotes okay? (I'm not a *nix expert so I don't know.)

Reply:
should be OK, but i'll take them out to avoid any issues. (I'll leave the © symbol in.)